frontend/src/utils/sentry-config.ts

```diff
@@ -15,8 +15,8 @@ export const sentryConfig: ModuleConfiguration = {
   logMockCalls: false,
   lazy: true,
   clientConfig: {
-    // Only allow errors that come from an actual openverse.engineering subdomain
-    allowUrls: [/^https?:\/\/(.*)\.openverse\.engineering/],
+    // Only allow errors that come from openverse.org or a subdomain
+    allowUrls: ["https://openverse.org", /^https?:\/\/(.*)\.openverse\.org/],
```

Comment:
The regex can include the first case.

```suggestion
    allowUrls: [/^https?:\/\/((.*)\.)?openverse\.org/],
```

Reply:
According to https://regex101.com/, @dhruvkb's suggestion accepts the base URL plus strings like:
- https://staging.openverse.org
- https://api-staging.openverse.org
Is it something else missing?

Reply:
🤦 I had a typo in 'openverse' in one of my domains! Thanks for testing!